docs/manuals/gui/index.md

```diff
@@ -45,3 +45,4 @@ content sent back to the client so the user can see the application interface
 and start using it.
 
 !!! info "You can find more information on how pages are created and used in Taipy application in the [Pages](pages.md) section."
+!!! info "For running Taipy Gui service with some other Taipy services, please refer to page [Running Taipy services](../running_services/index.md)"
```

Comment:
"For running Taipy Gui service"
->
"To run the Taipy GUI service"

"to page [Running Taipy services](https://github.com/Avaiga/taipy-doc/pull/running_services/index.md)"
to
"to the [Running Taipy services](https://github.com/Avaiga/taipy-doc/pull/running_services/index.md) page"